static/app/components/smartSearchBar/utils.tsx

```diff
@@ -695,7 +695,8 @@ export function getAutoCompleteGroupForInvalidWildcard(searchText: string) {
 
 export function escapeTagValue(value: string): string {
   // Wrap in quotes if there is a space
-  return value.includes(' ') || value.includes('"')
+  const isArrayTag = value.startsWith('[') && value.endsWith(']') && value.includes(',');
```

Comment:
Maybe this isn't an issue, but if any tags had autocomplete values that looked like an array (but were not) this would break those, right?

Reply:
yes it would